src/ApiV2.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import bodyParser from 'body-parser';
+import { Router } from 'express';
+
+import AppDevAPI from './appdev/AppDevAPI';
+import AlertsRouter from './routers/AlertsRouter';
+import AllStopsRouter from './routers/AllStopsRouter';
+import DelayRouter from './routers/DelayRouter';
+import HelloWorldRouter from './routers/HelloWorldRouter';
+import PlacesAutocompleteRouter from './routers/PlacesAutocompleteRouter';
+import RouteRouterV2 from './routers/RouteRouterV2';
+import MultiRouteRouter from './routers/MultiRouteRouter';
+import TrackingRouter from './routers/TrackingRouter';
+
+class APIv2 extends AppDevAPI {
```

Comment:
this is inconsistent with the filename, you should stick with either `ApiV2` or `APIv2` but not have both!